app/experimenter/nimbus-ui/src/components/PageRequestReview/index.tsx

```diff
@@ -127,34 +185,17 @@ const PageRequestReview = ({
                 </p>
               </Alert>
             )}
-            {status.draft &&
-              (showLaunchDraftToReview ? (
-                <FormLaunchDraftToReview
-                  {...{
-                    isLoading: loading,
-                    onSubmit: onLaunchClicked,
-                    onCancel: toggleShowLaunchDraftToReview,
-                    onLaunchToPreview: onLaunchToPreviewClicked,
-                  }}
-                />
-              ) : (
-                <FormLaunchDraftToPreview
-                  {...{
-                    isLoading: loading,
-                    onSubmit: onLaunchToPreviewClicked,
-                    onLaunchWithoutPreview: toggleShowLaunchDraftToReview,
-                  }}
-                />
-              ))}
+
             {status.preview && (
               <FormLaunchPreviewToReview
                 {...{
-                  isLoading: loading,
+                  isLoading: isLoading,
```

Comment:
some sugar could be simply `isLoading,`

Reply:
Ah, yeah, missed this when I renamed some symbols